torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -2233,6 +2254,67 @@ bool removeInplaceOperations(const std::shared_ptr<Graph>& graph) {
       graph, [&](Node* node) { return inplace_ops.count(node->kind()) != 0; });
 }
 
+// Recursively traverse blocks, gather all nodes with given symbol,
+// and then apply mutator function.
+void mutateNode(
```

Comment:
nitpick: this seems to be a general version of `traverseProfileIValues`. can we merge the two?